lib/SILOptimizer/PassManager/PassPipeline.cpp

```diff
@@ -101,7 +101,7 @@ SILPassPipelinePlan::getDiagnosticPassPipeline(SILOptions Options) {
   // disabled.
   if (Options.DebugSerialization) {
     addMandatoryDebugSerialization(P);
-    return P;
+    return const_cast<const SILPassPipelinePlan&>(P);
```

Comment:
I don't see any obvious reason to keep it. Pinging @gottesmm since he put it there.